bedrock/firefox/templates/firefox/family/index-b.html

```diff
@@ -0,0 +1,173 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import fxfamilynav with context %}
+
+{% extends "base-resp.html" %}
+
+{% block experiments %}
+  {% if LANG == 'en-US' %}
+    {% javascript 'experiment_firefox_family' %}
+  {% endif %}
+{% endblock %}
+
+{% block page_title_prefix %}{% endblock %}
+{% block page_title %}{{_('Free Web Browser for Android, iOS, Desktop | Firefox')}}{% endblock %}
+{% block page_desc %}{{_('Millions of people around the world trust Firefox Web browsers on Android, iOS and desktop computers. Fast. Private. Download now!')}}{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_family_index_b' %}
+{% endblock %}
+
+{% block body_id %}firefox{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block site_header_unwrapped %}
+  {% call fxfamilynav('firefox') %}
+    {{ download_firefox(dom_id="sticky-download-desktop", alt_copy=_('Download')) }}
+  {% endcall %}
+{% endblock %}
+
+{% block content %}
+
+<section class="content-section" id="intro">
+  <div class="content">
+    <h2>
+      {{ _('Firefox Web Browser: <br>Fast. Private. Safe.') }}
+    </h2>
+
+    <p>
+    {% trans %}
+      Firefox is faster, more private, and always independent. <br>Available on
+      desktop and mobile.
+    {% endtrans %}
+    </p>
+
+    {{ download_firefox(dom_id="download-firefox") }}
+  </div>
+</section>
+
+<section class="content-section" id="fast">
+  <div class="content">
+    <div class="copy">
+      <h3>{{ _('Browse faster with Firefox') }}</h3>
+
+      <p>
+      {% trans %}
+        Firefox is faster than ever before, thanks to a new multi-process
+        architecture that improves browser speed and page-load time.
+      {% endtrans %}
+
+      {% trans url1=url('mozorg.home'), url2='https://blog.mozilla.org/firefox/tag/quantum' %}
+        <a href="{{ url1 }}">Mozilla</a>, the nonprofit that builds Firefox,
+        invented a new programming language to
+        <a rel="external" href="{{ url2 }}">make Firefox faster</a>.
+      {% endtrans %}
+
+      {% trans %}
```

Comment:
Does each sentence need to be a separate trans block? Seems like the paragraph could just be one?